### DISCUSSION TITLE
Wow and Wonder Post for this week

### DISCUSSION THREAD

**Discussion by kaylakrug** [cmadland/edci338-202201#96] [Q&A] [answered]
I noticed on the course site you said post your sixth Wow and Wonder post, but I looked back and I think it's only the fifth one. Just want to make sure I have not missed one? I don't believe we had one to post for the first week.

**Comment by kaylakrug** [comment]
@cmadland

**Comment by cmadland** [comment, marked as answer]
That's correct...should have been number 5. Thanks. @kaylakrug
